ledger/internal/eval.go

```diff
@@ -1015,8 +1015,8 @@ func (eval *BlockEvaluator) checkMinBalance(cow *roundCowState) error {
 		dataNew := data.WithUpdatedRewards(eval.proto, rewardlvl)
 		effectiveMinBalance := dataNew.MinBalance(&eval.proto)
 		if dataNew.MicroAlgos.Raw < effectiveMinBalance.Raw {
-			return fmt.Errorf("account %v balance %d below min %d (%d assets)",
-				addr, dataNew.MicroAlgos.Raw, effectiveMinBalance.Raw, dataNew.TotalAssets)
+			return fmt.Errorf("account %v balance %d below min %d (%d assets) (%d boxes)",
```

Comment:
@algoidurovic For my understanding:  Did the change stem from debugging or for another reason?

Aside:  There's other data points _not_ included (e.g. `TotalBoxBytes`).

Reply:
Stemmed from debugging because the number of assets isn't the only variable used to determine minimum balance. If you'd like I can add the other variables as well.

Reply:
Open to thoughts from you + reviewers - My default position is to enrich error messages with available context.  Here, that default implies adding available fields impacting min balance.

Reply:
I think there's so many inputs to min balance that showing them all is a bit much, just in terms of the format string.  But maybe it would be reasonable to use "%+v" and print the whole `dataNew.AccountBaseData` object. Basically:
```
return fmt.Errorf("account %v is below min balance of %d. Details: %+v", addr, effectiveMinBalance.Raw, dataNew.AccountBaseData")
```